charts/kong/templates/deployment.yaml

```diff
@@ -123,6 +123,9 @@ spec:
       {{- if .Values.deployment.kong.enabled }}
       - name: "proxy"
         image: {{ include "kong.getRepoTag" .Values.image }}
+        {{- if .Values.command }}
+              command: {{ .Values.command }}
+        {{- end }}
```

Comment:
- If we go ahead with this change I'd suggest to use a more nested key name, e.g. `.Values.deployment.command`
- Please add this commented out to default `values.yaml`, and comment what this can be used for.

Reply:
Updated to use the more nested key name, and have also added this into the values.yaml file commented out. I've also fixed the indentation issue on L127